core-web/libs/ui/src/lib/components/dot-drop-zone/dot-drop-zone.component.ts

```diff
@@ -44,7 +44,7 @@ export class DotDropZoneComponent {
      * Max file size in bytes.
      * See Docs: https://www.dotcms.com/docs/latest/binary-field#FieldVariables
      */
-    @Input() maxFileSize: number;
+    @Input() maxFileSize: number | null = null;
```

Comment:
Question, this can be a default max file size, intead of null? Maybe exist a BE level a max file size defined.

Reply:
By default, there is no limit for the maximum file size. When it is set to null, there is no restriction